### ISSUE TITLE
add arabic language support !

### ISSUE THREAD

**Issue by brahimjarallah** [kanboard/kanboard#5396]
Please add the **missing arabic lanaguage** support to the kanboard app. 
regarrds.👍🏻

**Comment by imfx77** [comment]
@brahimjarallah 
Good luck with that ;)

If a software wasn't designed at the very beginning with the option to display right-to-left languages in mind, it is next to impossible to integrate that on a later stage when content and features have exploded. Or when it has entered maintenance mode, as has KB. Not to mention the numerous plugins that are developed independently of the KB itself, that will render totally incompatible with RTL language.

Furthermore, being a long time developer for a product that supports multiple weird languages - I can claim that Arabic is the most problematic of all: in means of grammar and word ordering rules, also mixing text and numbers and special symbols, and especially mixing Arabic with any other languages - all of which happens on a general every day basis for a collaborative software as KB. Even Chinese, Japanese, Korean, Vietnamese, Taiwanese, etc., which are traditionally written vertically, have an option for a horizontal form with a dominant left-to-right direction and straight word order. Not Arabic, not Hebrew, not Farsi, etc. Sorry to state it like that, but those are absolute exceptions in language handling, and a total hell to implement fully and correctly.

In short - it is not as simple as just adding another language pack - a correct and meaningful implementation implies a total redesign, needs tons of rework, and I bet there is no payback worth that effort for KB at this moment.

Still, I am curious what would the KB maintainers say about that ...
Yet, again, good luck - you'll need it ;)
